requirements.txt

```diff
@@ -0,0 +1,8 @@
+arrow
+bleach
+fedmsg[all]
```

Comment:
After a second look I believe this can get away with just depending on ``fedmsg[consumers]`` since I don't see any hard requirements on ``fedmsg.crypto``. I'll change it.